ckanext/publications_qld_theme/templates/header.html

```diff
@@ -35,6 +35,9 @@
         {% endblock %} {% block header_account_log_out_link %}
           {{ super() }}
         {% endblock %} {% endblock %}
+        {% if h.is_datarequests_enabled() %}
```

Comment:
not required for publications (unsure if anyone would read the requests on pub)

Reply:
Just making it match Data. It should be harmless. (And theoretically it's not impossible to include Data Requests in the future.)